SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1280,7 +1280,18 @@ void USpatialActorChannel::SendPositionUpdate(AActor* InActor, Worker_EntityId I
 {
 	if (InEntityId != SpatialConstants::INVALID_ENTITY_ID && NetDriver->HasServerAuthority(InEntityId))
 	{
-		Sender->SendPositionUpdate(InEntityId, NewPosition);
+#if !UE_BUILD_SHIPPING
+		if (!NetDriver->Connection->GetView()[EntityId].Authority.Contains(SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID))
+		{
+			UE_LOG(LogSpatialActorChannel, Verbose,
+				   TEXT("Trying to send Position component update but don't have authority! Update will not be sent. Entity: %lld"),
+				   EntityId);
+			return;
```

Comment:
NetDriver->HasServerAuthority checks this already, why do it again?

Reply:
blindly moved what was inside sender to here. Thanks for calling out